sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -1162,8 +1161,9 @@ private void executeQuery(
 
     private JobConfigurationQuery createBasicQueryConfig() {
       return new JobConfigurationQuery()
-          .setQuery(query.get())
           .setFlattenResults(flattenResults)
+          .setPriority("BATCH")
```

Comment:
createBasicQueryConfig() is also called in createReader(), dryRunQueryIfNeeded(), and executeQuery().
BigQuery defaults to iteractive if priority is not set. So, this PR will change createReader() and dryRunQueryIfNeeded() to batch. And, BATCH queries take minutes to hours to start based on the big query web doc:

batch query starts as soon as idle resources are available, usually within a few minutes. If BigQuery hasn't started the query within 24 hours, BigQuery changes the job priority to interactive.
https://cloud.google.com/bigquery/querying-data

IMO, createReader(), dryRunQueryIfNeeded() benefits from low latency, but executeQuery() doesn't. So, I like the current priority settings, as long as batch queries don't take hours to start.